### PR TITLE
Animas combo bolus tooltip

### PR DESCRIPTION
**Pull request by krystophv**
Updates to bolus tooltip for Animas combo boluses to properly present the fact that Animas does not report the combo bolus upfront/over time split.

### CODE REVIEW

**Comment by krystophv** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;pSVz2Kw3&#x2F;10-ux-1-dev-1-tst-1-blip-daily-new-combo-bolus-tooltip-for-animas"> &lt;ux:1, dev:1, tst:1&gt; [blip] [daily] new combo bolus tooltip for Animas</a>

**Comment by krystophv** [comment]
ping @jebeck for PR review

File: js/plot/util/drawbolus.js [hunk 1]
```diff
@@ -390,6 +390,8 @@ module.exports = function(pool, opts) {
       html: function(group, d) {
         var bolus = pluckBolus(d);
         var justBolus = (bolus.normal === commonbolus.getMaxValue(d)) && !d.carbInput;
+        var isAnimasCombo = bolus.annotations && Array.isArray(bolus.annotations) &&
+          bolus.annotations.length > 0 && bolus.annotations[0].code === 'animas/bolus/extended-equal-split';
```
Comment (jebeck) [inline js/plot/util/drawbolus.js]: This isn't going to work if the annotation isn't the first. In my data I know for a fact there are some where there's also an annotation re: unknown time of bolus cancellation. More robust method for searching for the annotation please?

File: js/plot/util/drawbolus.js [hunk 2]
```diff
@@ -435,7 +437,7 @@ module.exports = function(pool, opts) {
             .text(format.tooltipValue(commonbolus.getRecommended(d)));
         }
         // only show programmed when different from delivery
```
Comment (jebeck) [inline js/plot/util/drawbolus.js]: Perhaps update this comment?

**Comment by jebeck** [comment]
Just one TODO here re: searching for annotations more robustly, otherwise LGTM. @krystophv perhaps we could walk through how to publish tideline to npm when you've made the required change and are ready to merge? Then you'll need to also update blip for the new published version and tag.

**Pull request merged**